optimum/neuron/utils/misc.py

```diff
@@ -543,7 +551,11 @@ def replace_weights(
         weights = weights.state_dict()
 
```

Comment:
`from torch_neuronx.xla_impl.data_parallel import DataParallel`

Reply:
I tried, but the doc builder will complain if we import directly `torch_neuronx`...